share/ipld/helpers.go

```diff
@@ -0,0 +1,12 @@
+package ipld
+
+import (
+	"fmt"
+)
+
+func SanityCheckNID(nID []byte) error {
+	if len(nID) != NamespaceSize {
+		return fmt.Errorf("expected namespace ID of size %d, got %d", NamespaceSize, len(nID))
+	}
+	return nil
+}
```

Comment:
It doesn't worth much if it has its own file. Also, ipld pkg is not the place it should be in anyway. Can you just inline it inside the service and remove it completely from GetSharesByNamespace. The service anyway does the check before passing it down to GetSharesByNamespsace.

Reply:
Good point, thank you. File is gone.